stream_alert/alert_processor/helpers.py

```diff
@@ -30,3 +31,81 @@ def elide_string_middle(text, max_length):
 
     half_len = (max_length - 5) / 2  # Length of text on either side.
     return '{} ... {}'.format(text[:half_len], text[-half_len:])
+
+
+def publish_alert(alert, output, descriptor):
```

Comment:
I know these are 'publishers' and all... but the calls below of `publication = publish_alert(...)` threw me for a second. `publish_alert` seems almost too close to `dispatch`.. would you be open to something a little less 'transactional', like `compose_alert` for this function name? (not super set on this, just thinking out loud).

Reply:
I see your point. The choice of verb `publish` implies we're actively *publishing it to somewhere*. Maybe `get_alert_publication` is too wordy. `compose_alert` sounds about right so I can change that